core/src/test/scala/unit/kafka/log/LogManagerTest.scala

```diff
@@ -242,7 +243,11 @@ class LogManagerTest {
   def testRecoveryDirectoryMappingWithRelativeDirectory() {
     logManager.shutdown()
     logDir = new File("data" + File.separator + logDir.getName)
-    logDir.mkdirs()
+    try {
+          Files.createDirectory(logDir.toPath())    
+      } catch {
+        case e: IOException => //this is good
```

Comment:
If you the exception to be ignored, the previous code is better. However, it seems to me that an exception should cause the test to fail, no?